unit-tests/heitech.zer0mqXt.core.tests/PubSubTests.cs

```diff
@@ -39,15 +40,44 @@ public async Task SimplePubSub_InProc()
         [Fact]
         public async Task Publish_without_server_throws()
         {
-            SocketConfiguration config = new ConfigurationTestData().GetSocketConfigInProc;
+            var config = new ConfigurationTestData().GetSocketConfigInProc;
             var sut = new PubSub(config);
 
             // Act
-            Func<Task> pub = async () =>  await sut.PublishAsync<Message>(new Message());
+            Func<Task> pub = async () => await sut.PublishAsync<Message>(new Message());
 
             // Assert
-            await Assert.ThrowsAsync<NetMQ.EndpointNotFoundException>(pub);
+            bool noException = false;
+            try
+            {
+                await pub();
```

Comment:
Down below we `Assert` that no exception has been thrown ...what do you think about just having `await pub();` without a try-catch block, since this would throw an exception which we do not expect already and would fail the test, while also telling us which exception was thrown?

Reply:
I guess then we have no assert at all? seems kinda strange, but I guess I give it a try.

Reply:
aaa - I wasn't paying attention here ... now I see